src/dharma/Assignment78.java

```diff
@@ -0,0 +1,37 @@
+package dharma;
+
+public class Assignment78 {
+
+	public static void main(String[] args) {
+		
+		for (int i = 0; i < 4; i++) {
```

Comment:
Make code generic so if you want to print for 10 or 2 rows then also it is possible